riscv_private.h

```diff
@@ -322,13 +323,13 @@ static inline uint32_t calc_fclass(uint32_t f) {
   /* 0x020    rs1 is positive subnormal */
   out |= (!expn && frac && !sign) ? 0x020 : 0;
   /* 0x040    rs1 is positive normal */
-  out |= (expn && expn < 0x78000000 && !sign) ? 0x040 : 0;
+  out |= (expn && (expn != FMASK_EXPN) && !sign) ? 0x040 : 0;
   /* 0x080    rs1 is +INF */
-  out |= (f == 0x7f800000) ? 0x080 : 0;
+  out |= (expn == FMASK_EXPN && !frac && !sign) ? 0x080 : 0;
```

Comment:
Shall we update the description of these comments?

Reply:
How should I update the description?

Reply:
Explicitly mentioning signedness checks would be better.